client/apps/edit/components/header/index.coffee

```diff
@@ -51,6 +51,8 @@ module.exports = class EditHeader extends Backbone.View
 
   save: (e) ->
     e.preventDefault()
+    # Handles race condition to set props in a section that hasn't been clicked out of
+    $('.edit-section-container-bg')[0]?.click()
```

Comment:
There was a race condition where the 'save' click gets handled before we unmount from the component. This just clicks the container to unmount before the article is saved.